### ISSUE TITLE
[Provider] Uploading csv results in  Encoding::UndefinedConversionError ('"\xC3" from ASCII-8BIT to UTF-8') 

### ISSUE THREAD

**Issue by 1000miles** [sozialhelden/wheelmap#611]
**Steps to reproduced:**

- I am logged into the admin dashboard `/admin`
- I go to '/admin/provided_pois/' and click on the button `Upload CSV`
- I am on the page `/admin/provided_pois/import_csv`
- I chose an existent premium provider and the csv to be uploaded
- I click on the `Upload` button
- **Expected behavior**: The records from the csv should be successfully imported and I should get a success message with uploaded provided pois.
- **Actual behavior**:
  I get a `500 Internal Error` but under the hood I get a `"\xC3" from ASCII-8BIT to UTF-8` encoding error 

=> This behavior occurs currently in **locally** and **production** environment.

```
I, [2017-05-15T11:38:44.929005 #20910]  INFO -- : Completed 500 Internal Server Error in 14ms (ActiveRecord: 2.0ms)
F, [2017-05-15T11:38:44.937690 #20910] FATAL -- :
Encoding::UndefinedConversionError ("\xC3" from ASCII-8BIT to UTF-8):
  app/models/provided_poi.rb:13:in `import'
  app/admin/provided_poi.rb:27:in `block (2 levels) in <top (required)>'
  lib/rack_i18n_locale_switcher.rb:29:in `call'
  librato-rack (0.6.0) lib/librato/rack.rb:94:in `process_request'
  librato-rack (0.6.0) lib/librato/rack.rb:64:in `call'
```

Screenshot:

<img width="1262" alt="00000314" src="https://cloud.githubusercontent.com/assets/4077916/26196727/1af1d318-3bc0-11e7-92c1-1f47c92d05b9.png">

**Comment by 1000miles** [comment]
**Referencing Github issue:**
NoMethodError in Admin::ProvidedPoisController#import_csv #252

**Comment by 1000miles** [comment]
Fixed in PR #613. Closing.
